packages/web/public/static/locales/de/notifications.json

```diff
@@ -0,0 +1,22 @@
+{
+  "feedTitle": "Benachrichtigungen",
+  "emptyFeed": "Keine Benachrichtigungen ... bis jetzt!",
+  "levelOne": {
+    "newFollowers": "Du hast {{followerCount}} neue Follower",
+    "threadComments": "Du hast {{commentCount}} neue Feedback-Kommentare zu diesem Beitrag",
+    "postComments": "Du hast {{commentCount}} neue allgemeine Kommentare zu diesem Beitrag",
+    "postClaps": "{{clapCount}} Leute haben diesem Beitrag applaudiert",
+    "threadCommentThanks": "{{userIdentifier}} hat für {{thanksCount}} Feedback-Kommentare Danke gesagt",
+    "newPosts": "Journaler, denen du folgst, haben {{postCount}} neue Beiträge veröffentlicht"
+  },
+  "levelTwo": {
+    "newFollowers": "Diese {{followerCount}} großartigen Journaler folgen dir nun",
+    "threadCommentThanks": "Du hast {{thanksCount}} Danke für Feedback zu diesem Beitrag erhalten",
+    "postClaps": "Diese {{userCount}} Leute haben deinem Beitrag applaudiert",
+    "newPosts": "New posts from your community"
```

Comment:
@LindasLinguas line 16 here is new 😊🙏🏼

Reply:
```suggestion
    "newPosts": "Neue Beiträge aus deiner Community"
```

Reply:
Community normally would translate to "Gemeinschaft", but in my eyes, this doesn't really fit in Journaly's context... That's why I kept the English word "Community".